change/@fluentui-eslint-plugin-54a8dcfd-724c-4111-a35e-1320a6dd6cae.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: create core base for react/node/legacy lint configs",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "none"
```

Comment:
Awesome that you set `dependentChangeType` to `none`

Reply:
learned my lesson from the best ;)